docs/user/intro.rst

```diff
@@ -38,10 +38,11 @@ But there is more.. you can also give your module a dictionary in an interactive
 
 ::
 
-    >>> from mymodule import MyModule
+    >>> from argschema import ArgSchemaParser
+    >>> from mymodule import MySchema
     >>> d = {'a':5}
-    >>> mod = MyModule(input_data=d)
-    >>> mod.run()
+    >>> mod = ArgSchemaParser(input_data=d,schema_type=MySchema)
+    >>> print mod.args
```

Comment:
Let us just write portable Python from the start on everything, so this should be print(mod.args)